pylint_odoo/checkers/modules_odoo.py

```diff
@@ -366,11 +366,14 @@ def check_odoo_relative_import(self, node):
 
     def check_folder_test_imported(self, node):
         if (hasattr(node.parent, 'file')
-            and os.path.basename(node.parent.file) == '__init__.py'
-            and ("tests" in [item for item, _ in node.names]
-                 or "tests" in node.modname)):
-            self.add_message('test-folder-imported', node=node,
-                             args=(node.parent.name,))
+                and os.path.basename(node.parent.file) == '__init__.py'):
+            if isinstance(node, astroid.ImportFrom):
```

Comment:
What about initialize the variable `package_names = []`

In order to avoid the following error: `NameError: name 'package_names' is not defined` if the method is called using a node different to `*Import` classes

```suggestion
                and os.path.basename(node.parent.file) == '__init__.py'):
            package_names = []
            if isinstance(node, astroid.ImportFrom):
```

Reply:
Done